lib/canvas/live_events.rb

```diff
@@ -101,6 +101,23 @@ def self.get_submission_data(submission)
     }
   end
 
+  def self.get_attachment_data(attachment)
+    {
+      attachment_id: attachment.global_id,
+      user_id: attachment.global_user_id,
+      display_name: LiveEvents.truncate(attachment.display_name),
+      filename: LiveEvents.truncate(attachment.filename),
+      context_type: attachment.context_type,
+      context_id: attachment.context_id,
+      content_type: attachment.content_type,
+      folder_id: attachment.folder_id,
```

Comment:
On a slightly tangential note, we're currently only interested in attachment events that take place on attachments in courses that are exposed in the files tool. Meaning:
 - files that get uploaded in the files tool
 - files that get uploaded as a discussion, assignment, .. attachment and end up at `unfiled` (or somewhere exposed in the files tool).

Is it safe to assume that an attachment with `context_type = 'Course'` always gets exposed in the course files tool? AFAICT, submissions are recorded with a `User` context, but I might not get exposed to all the places that have attachments logic.

Our current plan is to perform an extra API request to `/api/v1/folders/:folderId` and double check the name of the folder starts with `course files`. It would be nice not to have to do this.